### ISSUE TITLE
[BUG]Regarding the function under dynamics._extend_centers_gpu

### ISSUE THREAD

**Issue by Pazen-ai** [MouseLand/cellpose#1001]
In this function, `T[tuple(neighbors[:, 0])] = Tneigh.mean(axis=0)` there seems to be something wrong with this line of code.
`neighbors.shape`  is (2,9,N) due to `neighbors = torch.stack((neighborsY, neighborsX), dim=0)` but [:,0] will create (2,N) result.
This does not match the formatting that follows.

**Comment by carsen-stringer** [comment]
this is correct, the pixel is set to the mean of the values of its neighbors, can you please attach the error message?

**Comment by Pazen-ai** [comment]
sorry, I don't have the error message because I just read this part, so I reported it

**Comment by Pazen-ai** [comment]
> sorry, I don't have the error message because I just read this part, so I reported it

maybe I'm wrong ,cellpose works well on my pictures

**Comment by carsen-stringer** [comment]
yes this is the correct equation
